### ISSUE TITLE
Do we need to explain shared forms in the intro to 05-forms.md

### ISSUE THREAD

**Issue by rhuffstedtler** [mikecrittenden/drupal-7-deconstructed#20]
At line 164 we mention how we handle inheriting validation for shared forms, but we've never mentioned that concept before. Do we want to assume that the reader knows about forms inheritance in Drupal, or should we briefly say something in the intro.

I lean toward the latter option, because it isn't easy to come up with a google search that gives a good description of shared forms / forms inheritance.

**Comment by crittermike** [comment]
Yeah - that's probably a good idea. Probably just a few sentences would suffice, don't you think?
